pages/bookmark/index.js

```diff
@@ -0,0 +1,34 @@
+import { StyledUl } from "@/components/ActivitiesList";
+import { StyledListItem } from "@/components/ActivitiesList";
+import ActivityCardPreview from "@/components/ActivityCardPreview";
+import Link from "next/link";
+import Navigation from "@/components/Navigation";
+
+export default function BookmarkList({ activitiesInfo }) {
+  const filterBookmarks = activitiesInfo.filter(
+    (activityInfo) => activityInfo.isBookmarked === true
+  );
+
```

Comment:
you should check the filterBookmarks.length here and react with an early return statement.